daq_main2.py

```diff
@@ -27,6 +28,13 @@
 logger.addHandler(handler1)
 logger.addHandler(handler2)
 
+if not getpass.getuser().startswith("lsdc-"):
```

Comment:
Is the set of allowed accounts very large? Could they be spelled out fully and checked for an exact match, to future-proof against possible other lsdc-related accounts being accidentally used here?

Reply:
there are currently only 3 "lsdc-" service accounts. I did not want to hard-code these into the code but would it be more satisfactory to put them in here? I will need to think about how best to do this.

Reply:
I think that's safer. We add beamlines rarely, and this will avoid potential unintended permissiveness.

Reply:
fixed in https://github.com/NSLS-II/lsdc/pull/282/commits/be9ddd08337e4be053ee7ff6b8b012b0e4dfe77d . if you are happy with this fix, please resolve!